scripts/doctor

```diff
@@ -42,7 +42,16 @@ else
     count_in_path "bundler"
     if [ $found -eq 0 ]; then
         exitcode=1
-        warn "yel" "   bundler not found. Run 'gem install bundler' to install it it."
+        warn "yel" "   bundler not found. Run 'gem install bundler' to install it."
+    else
+        warn "grn" "   found!"
+    fi
+    count_in_path "bundle"
+    if [ $found -eq 0 ]; then
+        exitcode=1
+        warn "yel" "   bundler not found. Run 'gem install bundler' to install it."
+    elif [ $found -gt 1 ]; then
+        warn "yel" "   too many bundle found. You may have something overriding 'bundle'."
```

Comment:
after some testing with this.. Let's omit this conditional. For me `bundle` is defined in multiple places on PATH as is.  In ruby shim definitions and in bin so it's causing unnecessary noise. 

Another way to go about this would be to check the output of the binary. For example `bundle version` prints the keystring `Bundler version`.. Which you could check for.

Reply:
It's how I would have done it at first but chose to just be consistent  with the other checks!